src/gui/tray/usermodel.cpp

```diff
@@ -183,7 +183,12 @@ void User::showDesktopTalkNotification(const Activity &activity)
 {
     const auto notificationId = activity._id;
 
-    if (!canShowNotification(notificationId)) {
+    const ConfigFile cfg;
+    const auto userStatus = _account->account()->userStatusConnector()->userStatus().state();
+    if (!canShowNotification(notificationId) ||
+        userStatus == OCC::UserStatus::OnlineStatus::DoNotDisturb ||
+        !cfg.showChatNotifications()) {
+
```

Comment:
the local check of DnD mode is clearly duplicating a check made server side
is there any reason to add it there ?
seems like redundant and adding one more check in the if for nothing

Reply:
I was likely being cautious here

I removed the check